app/models/school.rb

```diff
@@ -85,4 +101,21 @@ def invite_school_contact
       false
     end
   end
+
+  def what_to_order_text
```

Comment:
Again, not sure about putting this logic on the model – I'd be ok with this living in the view.

Reply:
^ Seconded 👍  Let's try to keep presentation code out of the models, and either in the views (if it's simple) or helpers/components (if it's more complex)

Reply:
Moved this one to the view helper